src/javascript/JContent/ContentRoute/ToolBar/FileModeSelector/FileModeSelector.jsx

```diff
@@ -65,7 +54,7 @@ export const FileModeSelector = () => {
         }
     };
 
-    let select = mode === GRID && gridMode === THUMBNAIL ? THUMBNAIL : (mode === GRID ? DETAILED : LIST_VIEW);
+    let select = mode === GRID && gridMode === THUMBNAIL ? THUMBNAIL : (mode === GRID ? THUMBNAIL : LIST_VIEW);
```

Comment:
With the current code :
```suggestion
    let select = mode === GRID ? THUMBNAIL : LIST_VIEW
```
Or with the simplification suggested above :
```suggestion
    let select = mode 
```